CONTRIBUTING.md

```diff
@@ -3,7 +3,8 @@
 We're always looking for your help to improve the product (bug fixes, new features, documentation, etc).
 
 ## Contribute a code change
-* Start by reading the [Engineering Design](https://onnxruntime.ai/docs/reference/high-level-design).
+
+* Start by reading the [Engineering Design](https://onnxruntime.ai/docs/reference/high-level-design.html). More documentation can be found in the [docs folder](./docs/) and [on the docs website](https://onnxruntime.ai/docs).
```

Comment:
Is there a distinction to call out between the docs hosted on github vs gh-pages? We also have documentation in the wiki: https://github.com/microsoft/onnxruntime/wiki (developer-oriented)

In general, I think the ones in github (both /docs and the wiki) are more relevant to developers of ORT, while the ones in gh-pages are for consumers of ORT. If that's accurate, it may be useful to call this out (and also ensure docs are located as described).

Reply:
Sure - clarified  the wording - let me know what you think